packages/apps-config/src/api/spec/index.ts

```diff
@@ -221,6 +222,7 @@ const spec: Record<string, OverrideBundleDefinition> = {
   'jur-node': jur,
   kerria: parallel,
   khala,
+  ...kilt,
```

Comment:
I think this might be a mistake since we want the key to be `kilt`, unless the object passed from `kilt.js` is `{ kilt: {} }`

Reply:
I've based this on acala's approach. The kilt object contains a key per runtime - we've got spiritnet, the production network, and several test networks with separate runtime/spec names. I tested this locally and it seems to work as expected.

Reply:
Ahh okay thanks for the explanation, everything looks good then!